Source/multi.cpp

```diff
@@ -708,6 +712,8 @@ void NetClose()
 	SNetLeaveGame(3);
 	if (gbIsMultiplayer)
 		SDL_Delay(2000);
+	Players.clear();

```

Comment:
This is the cause of the failing test. The player state is being wiped before the test makes the comparison between game state and reference save.